### PR TITLE
mv, whoami, whois: add Chinese translation

### PR DESCRIPTION
**Pull request by HugueLiu**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/mv.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# mv
+
+> 移动或重命名文件或目录。
+> 更多信息：<https://www.gnu.org/software/coreutils/mv>.
+
+- 移动文件到任意位置：
+
+`mv {{source}} {{target}}`
```
Comment (blueskyson) [inline pages.zh/common/mv.md:8]: Please translate all the tokens as well.
```suggestion
`mv {{来源}} {{目标}}`
```

File: pages.zh/common/mv.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- 移动文件到其他目录，并保持文件名不变：
+
+`mv {{source1}} {{source2}} {{source3}} {{target_directory}}`
```
Comment (blueskyson) [inline pages.zh/common/mv.md:12]:
```suggestion
`mv {{来源1}} {{来源2}} {{来源3}} {{目标目录}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution, here are some suggestions.

File: pages.zh/common/mv.md (hunk 2 above)
Comment (HugueLiu) [reply pages.zh/common/mv.md]: Thanks for reply, the problems have been resolved.

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**